src/utils/index.test.ts

```diff
@@ -1,7 +1,7 @@
 import { convertSecondsToMinAndSec, isEthereumAddress } from "./";
 
 describe("utils/isEthereumAddress", () => {
-  test("should work", () => {
+  it("should work", () => {
```

Comment:
why?

Reply:
after extending jest's expect, some weird type errors come about for this particular test, just changed it to `it` to get rid of the errors

Reply:
it looks like testcafe's `test` is interfering with the types here, yeah `it` should be fine since it's supposed to be jest